packages/react-refresh/src/ReactFreshRuntime.js

```diff
@@ -435,6 +435,7 @@ export function injectIntoGlobalHook(globalObject: any): void {
       // Otherwise, the renderer will think that there is no global hook, and won't do the injection.
       let nextID = 0;
       globalObject.__REACT_DEVTOOLS_GLOBAL_HOOK__ = hook = {
+        renderers: new Map(),
```

Comment:
This is just so the lines below don't crash when we take this code branch. This particular Map isn't actually used further.